### ISSUE TITLE
Pull to Refresh

### ISSUE THREAD

**Issue by pwd-github** [Iamlooker/Droid-ify#215]
**Is your feature request related to a problem? Please describe.**
Please implement Pull to Refresh on the Installed and Updates page. 

**Describe the solution you'd like**
On the Installed and Updates page, we have to press all the way in the top right of the page to refresh. Please implement pull to refresh. 

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Comment by pbanj** [comment]
This is the one thing that keeps me from using this over the standard one

**Comment by peterge1998** [comment]
+1 👍

**Comment by Donkey-Doug** [comment]
Or just remove the refresh button and rely on push.

**Comment by pwd-github** [comment]
> Or just remove the refresh button and rely on push.

Yeah, definitely not. I press that button way too much to just passively rely on pushes.

**Comment by symbuzzer** [comment]
Still need this! Onlymiss on this app
